### DISCUSSION TITLE
Chainlink Hackathon, finding teammates

### DISCUSSION THREAD

**Discussion by ManiBharathiSS** [smartcontractkit/full-blockchain-solidity-course-py#1745] [General]
Hey guys,

 I am looking for a team for the Chainlink Hackathon. I have experience in solidity and brownie. I can put lot of time in the project. I am fully focusing on the hackathon right now, so if you want a serious teamate, i am in. 

This is my github :https://github.com/ManiBharathiSS

Lets build a great project together.

@Neftyr

**Comment by Niferu** [comment]
Hello @ManiBharathiSS 
We will be happy to work with you!

I'm currently working on project with @MarkizOrias and we are also looking for someone more skilled to join our group to get some experience. We can provide a lot of financial knowledge background as we are working professionally in this industry pretty long. We look forward for tomorrow's team matching to pick someone to our team :)

**Reply by ManiBharathiSS** [reply]
Cool. So how can i dm you? Are you in twitter or any other social media ?

this is my twitter id : @manibharathi_ss

**Reply by cromewar** [reply]
It's very exciting to see people from this repo team up to build amazing things. I would love to give a hand as well.

**Reply by ManiBharathiSS** [reply]
this is my discord id: mani_bharathi_ss#4126

**Reply by ManiBharathiSS** [reply]
oh thanks @cromewar . It would be fantastic if you could help us :)

**Reply by Niferu** [reply]
Hi @ManiBharathiSS 
I have just sent you friend request on discord, my nick there is Niferu :)
@cromewar it is great to hear that! I'm so hyped to work with you guys!

**Reply by cromewar** [reply]
cromewar#6054 my discord handle :)

**Reply by adityabhattad2021** [reply]
Hey is the team still open? would love to join

**Reply by Niferu** [reply]
Hello @adityabhattad2021 

We are currently looking only for Front-End developer to join our team.

**Reply by adityabhattad2021** [reply]
I am up...

**Reply by adityabhattad2021** [reply]
How can we connect @Neftyr?

**Reply by adityabhattad2021** [reply]
Sent you a DM, hoping you check it out.
